src/pages/user-profile/user-profile.tsx

```diff
@@ -0,0 +1,48 @@
+import { UserController } from 'networking/controllers/user-controller';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import styles from './profile.module.scss';
+
+const UserProfile = () => {
+  type Params = {
+    id: string,
+  };
+  const { id } = useParams<Params>();
+  const [user, setUser] = useState<FullDummyUser>();
+
+  useEffect(() => {
+    UserController.getUser(id).then((value) => {
+      setUser(value);
+    });
+  }, [id]);
+
+  return (
+    <>
+      <h1>User profile 👤</h1>
+      {!user && (
+        <p>Loading...</p>
+      )}
+      {user && (
+        <div className={styles.userInfo}>
+          <div className={styles.profilePicture}>
+            <img alt="profile" src={user?.picture} />
+          </div>
+          <div className={styles.wrapper}>
+            <p>Firstname:</p>
+            <p>{user.firstName}</p>
+            <p>Lastname:</p>
+            <p>{user.lastName}</p>
+            <p>Birthdate:</p>
+            <p>{new Date(user.dateOfBirth).toLocaleDateString('en-GB')}</p>
```

Comment:
I would create a new helper function called `formatDateOfBirth(user.dateOfBirth)`. 

Also, any particular reason why you chose `en-GB`? I think you can leave it empty and it will adapt to the user's locale. (I think it's fine to leave `en-GB` if you like that format though! :smile: )